wordpress-mobile/WordPress-iOS
# Make Plugin Directory performance better.

Fixes #8635.

To test:
1. Open the Plugin directory screen
2. See that the main thread no longer freezes for a second because of the HTML parsing.

Also:
1. Open the "WP Super Cache" plugin page
2. See that the the loading now doesn't take as long as before. It's still not as fast as it should be, but better than before, thanks to switching to UITextVIew.
